### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by caleb-eckenwiler**
I don't know what I'm doing, but I need a readme for the Assets and Vulns task.

### CODE REVIEW

File: tasks/connectors/veracode_asset_vulns/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The data is batched by Application before being sent to Kenna. 
+
+1. Pull a list of applications (https://help.veracode.com/r/c_apps_intro)
+1. Pull a list of assets and vulns for each application and submit to Kenna (https://help.veracode.com/r/c_findings_v2_intro)
+
+
+## Command Line
+
+
```
Comment (rick-kenna) [inline tasks/connectors/veracode_asset_vulns/readme.md:18]: Only need one blank line.

File: tasks/connectors/veracode_asset_vulns/readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| --- | --- | --- | --- |
+| veracode_id | true | Veracode ID | n/a |
+| veracode_key | true | Veracode API Key | n/a |
+| veracode_page_size | true | Number of records to bring back with each page request from Vercode | n/a |
+| kenna_api_key | false | Kenna API Key for use with connector option | n/a |
+| kenna_api_host | false | Kenna API Hostname if not US shared | api.kennasecurity.com |
+| kenna_connector_id | false | If set, we'll try to upload to this connector | n/a |
+| output_directory | false | If set, will write a file upon completion. Path is relative to #{$basedir} | output/veracode |
+
+Example Run:
```
Comment (rick-kenna) [inline tasks/connectors/veracode_asset_vulns/readme.md:31]: I would make "Example Run:" either a heading or in bold.

**Review by rick-kenna — APPROVED** [review]

**Comment by jcran** [comment]
Nice job @caleb-eckenwiler!

**Review by rick-kenna — APPROVED** [review]

**Comment by rick-kenna** [comment]
Thanks Caleb.

**Pull request merged**